test/integration/admin_can_modify_only_their_account_data_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'test_helper'
+
+class AdminCanModifyOnlyTheirAccountDataTest < ActionDispatch::IntegrationTest
+  test 'admin can modify their account data' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.